include/vapor/FlowParams.h

```diff
@@ -53,35 +53,94 @@ class PARAMS_API FlowParams : public RenderParams {
 
     static std::string GetClassType() { return ("FlowParams"); }
 
-    // True  == Steady; False == Unteady
+    //! Sets the type of flow rendering algorithm being used.
+    //! Steady flow (streamlines) renders time-invariant trajectories that follow a vector field at a single timestep.
+    //! Unsteady flow (pathlines) render time-variant trajectories that advect through the timeseries of a loaded dataset.
+    //! \param[in] bool - Steady/streamlines = 1, Unsteady/pathlines = 0
     void SetIsSteady(bool steady);
+    
+    //! Gets the type of flow rendering algorithm being used.
+    //! Steady flow (streamlines) renders time-invariant trajectories that follow a vector field at a single timestep.
+    //! Unsteady flow (pathlines) render time-variant trajectories that advect through the timeseries of a loaded dataset.
+    //! \retval bool - Steady/streamlines = 1, Unsteady/pathlines = 0
     bool GetIsSteady() const;
 
+    //! Get the multiplier being applied to the flow advection algorithm.
+    //! If there happens to be a mismatch between the units of your data's domain and the units of a variable such as wind speed,
+    //! you can scale the wind field with this parameter.  IE - If your data's domain is written in kilometers but your wind
+    //! vectors are in meters, you can apply a velocity multiplyer of 1000 to correct the mismatch.
+    //! \retval double - Velocity field multiplier for flow rendering
     double GetVelocityMultiplier() const;
+    
+    //! Set the multiplier being applied to the flow advection algorithm.
+    //! If there happens to be a mismatch between the units of your data's domain and the units of a variable such as wind speed,
+    //! you can scale the wind field with this parameter.  IE - If your data's domain is written in kilometers but your wind
+    //! vectors are in meters, you can apply a velocity multiplyer of 1000 to correct the mismatch.
+    //! \param[in] double - Velocity field multiplier for flow rendering
     void   SetVelocityMultiplier(double);
 
+    //! Get the number of steps to advect a steady flow line (aka a streamline)
+    //! \retval long - The number of steps a steady flow line is advected
     long GetSteadyNumOfSteps() const;
+    
+    //! Set the number of steps to advect a steady flow line (aka a streamline)
```

Comment:
```suggestion
    //! Set the number of integration steps to advect a steady flow line (aka a streamline). The integration step size is determined automatically and may vary over the integration
```

@shaomeng please verify that the above is correct.

Reply:
It is correct, but I feel that SetSteadyNumOfSteps() has a better explanation on this topic. I was expecting that one of the two has a more detailed documentation, which is the setter in this case, and the other one adopts \seealso or \copydoc.